src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -621,16 +619,10 @@ const formConfig = {
         },
       },
     },
-    additionalForms: {
-      title: 'Additional Forms',
+    mentalHealth: {
+      title: 'Mental health statement',
+      path: 'mental-health-form-0781',
       pages: {
-        additionalFormsChapterWrapper: {
-          title: 'Additional forms to support your claim',
-          path: 'additional-forms',
-          depends: formData => showAdditionalFormsChapter(formData),
```

Comment:
I believe the utility [`showAdditionalFormsChapter`](https://github.com/department-of-veterans-affairs/vets-website/blob/09d6337096b0680a1bd3434d03388aa6858bb9f1/src/applications/disability-benefits/all-claims/utils/index.jsx#L397) can be removed as it is only being called here.